src/uwtools/cli.py

```diff
@@ -164,11 +164,7 @@ def _add_subparser_config_validate(subparsers: Subparsers) -> SubmodeChecks:
     _add_arg_schema_file(required)
     optional = _basic_setup(parser)
     _add_arg_input_file(optional)
-    _add_arg_input_format(optional, choices=[FORMAT.yaml])
-    checks = _add_args_quiet_and_verbose(optional)
-    return checks + [
-        partial(_check_file_vs_format, STR.infile, STR.infmt),
-    ]
```

Comment:
Removed some currently unneeded complexity: The only configs we can currently validate are YAML ones, so for now simply insist that configs are YAML. If we need to re-complicate this later, we'll probably be working with more knowledge and will implement a better solution.